resources/local/chainspec.toml.in

```diff
@@ -189,7 +193,11 @@ max_timestamp_leeway = '5 seconds'
 
 [transactions.v1]
 # The maximum length in bytes of runtime args per V1 transaction.
-max_args_length = 1024
+# [0] -> Transaction Kind
+# [1] -> Max transaction size
+# [2] -> Max args length
+# [3] -> Transaction gas limit
```

Comment:
This needs a couple more fields. One is the lane transaction limit. The other is an annotation (i.e., "mint", "auction" and "install," nothing for general lanes). It doesn't make a lot of sense to allow assignment of arbitrary lane identifiers and some properties, but then also hardcode them (see lines 178-182 above).

The end result here should enable an arbitrary configuration of block structure, including disabling dedicated Mint/Auction lanes and having all lanes be install lanes. More importantly for our expected usage, it would make it trivial to tweak number and properties of general and install lanes.

Reply:
I don't know if it makes sense to do that, because 1.x Deploys count towards the large count, I've already expanded the limits to account for the new lane